src/Datadog.Trace/RuntimeMetrics/Meter.cs

```diff
@@ -0,0 +1,35 @@
+namespace Datadog.Trace.RuntimeMetrics
+{
+    internal class Meter
```

Comment:
I didn't see this type being used in this PR. Do you still need it?

Reply:
doh!